.github/workflows/response_templates/template_script.py

```diff
@@ -0,0 +1,116 @@
+import argparse
```

Comment:
lets add a short commented description of what this script does, consider adding example command line to execute this locally!